lib/cartopy/examples/__init__.py

```diff
@@ -0,0 +1,9 @@
+
+import sys
+
+import matplotlib
+
+
+# Set invisible backend for examples if running tests
+if 'nosetests' in sys.argv[0]:
```

Comment:
Is this still needed? If so, do we need to modify https://github.com/SciTools/cartopy/blob/master/lib/cartopy/tests/mpl/test_examples.py#L26?